_data/toc.yaml

```diff
@@ -1279,12 +1279,8 @@ manuals:
   section:
   - path: /compose/
     title: Overview of Docker Compose
-  - sectiontitle: Compose V2
-    section:
-    - path: /compose/cli-command/
-      title: Overview
-    - path: /compose/cli-command-compatibility/
-      title: Compose v2 compatibility
+  - path: /compose/cli-command-compatibility/
```

Comment:
@glours Suggest we move this to the end of the table of contents so new users aren't distracted by the info on compatibility. WDYT?